samples/nrf9160/serial_lte_modem/Kconfig

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-source "$ZEPHYR_BASE/Kconfig.zephyr"
```

Comment:
Is this change intended? The removed version looks fine?

Reply:
Yes, Nits issues before. "Redundant 'source "$(ZEPHYR_BASE)/Kconfig.zephyr" in 'samples/nrf9160/serial_lte_modem/Kconfig'. Just do 'source "Kconfig.zephyr"'
instead. The $srctree environment variable already points to the Zephyr root,
and all 'source's are relative to it."